react/src/lib/components/SubsurfaceViewer/layers/labeled_points/labeledPointsLayer.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import SubsurfaceViewer from "../../SubsurfaceViewer";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+export default {
+    component: SubsurfaceViewer,
+    title: "SubsurfaceViewer / LabeledPoints Layer",
+} as ComponentMeta<typeof SubsurfaceViewer>;
+
+const defaultParameters = {
+    docs: {
+        inlineStories: false,
+        iframeHeight: 500,
+    },
+};
+
+// Small example using labeledPointsLayer.
+const labeledPointsLayer = {
+    "@@type": "LabeledPointsLayer",
+    id: "labeledPoints-layer",
+
+    /*eslint-disable */
```

Comment:
Suggest fixing eslint condition or disabling only specific condition.

Reply:
Didn't manage to achive the array format desired. Tried eslint-disable array-element-newline with many different options.